docs/config-file/v2.rst

```diff
@@ -88,6 +88,10 @@ Example:
       # Build all formats
       formats: all
 
+.. note::
+
+   PDF/epub output is not supported when using MkDocs  
```

Comment:
I think that `htmlzip` is not supported as well. Can you confirm this?

Reply:
yupp, on it